### DISCUSSION TITLE
Unbound interstitial concentrations

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#1167] [Questions & Problems]
Hello, how are unbound interstitial concentrations of compounds computed in PK Sim? Is it using the interstitial fluid volume, or from free fraction and total organ concentration?

**Comment by StephanSchaller** [comment]
Hi Paul,

the interstitial space is its own compartment in PK-Sim. The output concentration will give you total concentration in interstitial Space, e.g.:
![image](https://user-images.githubusercontent.com/25477643/158056144-92e215b4-afa1-4b10-b757-c2f6c24f4c34.png)

However, there is no output for "unbound interstitial" (you could build an observer in MoBi, though).
"unbound interstitial" Could be derived by multiplying the default (total) interstitial concentration and multiplying it with the water/interstitial partition coefficient, which you can access through MoBi:
![image](https://user-images.githubusercontent.com/25477643/158056261-345c26e7-9652-4091-8c21-3c141f8adf5d.png)

**Comment by msevestre** [comment]
@StephanSchaller is correct. This is the way to calculate.
However, we already have an observer for you out of the box called "Interstitial Unbound". If you export to MoBi, you will see it under `Container Observer`

![image](https://user-images.githubusercontent.com/1041237/158067341-2cb920d3-3c4b-4c70-ad0d-517ed3f2bab6.png)

**Reply by pburkat** [reply]
Thank you both. In PK-Sim, there is an output called unbound interstitial, I have been using it all along. The reviewer for my paper inquired about how it is calculated, so this is very helpful.
